tubular/panel_backend_api.py

```diff
@@ -0,0 +1,26 @@
+import requests
+
+
+RETIREMENT_ENDPOINT = '/api/v1/retirement/'
+
+class PanelBackendAPI():
+    """
+    Panel Backend API to retire user data.
+    """
+
+    def __init__(self, panel_backend_base_url, panel_backend_auth_token):
```

Comment:
rename these variables and remove the unnecessary addition of `panel_backend`

Reply:
done